_includes/templates/download.html

```diff
@@ -0,0 +1,139 @@
+      <div>
+        <img src="/assets/images/os/med_win.png" alt="windows">
+        <span>
+          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.win64exe }}" id="downloadwinexe">Windows</a>
+          <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.win64exe }}" class="dl" id="win64exe">64 bit</a> -
+            <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.win32exe }}" class="dl" id="win32exe">32 bit</a></span>
+        </span>
+      </div>
+      <div>
+        <img src="/assets/images/os/med_win.png" alt="windows">
+        <span>
+          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.win64zip }}" id="downloadwinzip">Windows (zip)</a>
+          <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.win64zip }}" class="dl" id="win64zip">64 bit</a> -
+            <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.win32zip }}" class="dl" id="win32zip">32 bit</a></span>
+        </span>
+      </div>
+      <div>
+        <img src="/assets/images/os/med_osx.png" alt="osx">
+        <span>
+          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}{{ site.data.binaries.macdmg }}">Mac OS X</a>
+          <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}{{ site.data.binaries.macdmg }}" class="dl" id="macdmg">dmg</a> -
+            <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.mactar }}" class="dl" id="mactar">tar.gz</a></span>
+        </span>
+      </div>
+    </div>
+    <div>
+      <div>
+        <img src="/assets/images/os/med_linux.png" alt="linux">
+        <span>
+          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.lin64 }}" id="downloadlin">Linux (tgz)</a>
+          <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.lin64 }}" class="dl" id="lin64">64 bit</a> -
+            <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.lin32 }}" class="dl" id="lin32">32 bit</a></span>
+        </span>
+      </div>
+      <div>
+        <img src="/assets/images/os/arm.png" alt="ARM Linux">
+        <span>
+          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-arm-linux-gnueabihf.tar.gz" class="dl">ARM Linux</a>
+          	<span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-aarch64-linux-gnu.tar.gz" class="dl" id="lin64arm">64 bit</a> -
+            	<a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-arm-linux-gnueabihf.tar.gz" class="dl" id="lin32arm">32 bit</a></span>
+        </span>
+      </div>
+      <div>
+        <img src="/assets/images/os/med_ubuntu.svg" alt="ubuntu">
+        <span>
+          <a href="https://launchpad.net/~bitcoin/+archive/ubuntu/bitcoin">Ubuntu (PPA)</a>
+        </span>
+      </div>
+    </div>
+    <p class="downloadmore">
+      <a href="{{ PATH_PREFIX }}/SHA256SUMS.asc" class="dl">{{ page.downloadsig }}</a><br>
+      <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}.torrent" class="dl">{{ page.downloadtorrent }}</a>
```

Comment:
This is the same as the SHA256SUMS.asc file in the previous comment: that file only exists on the server because it is part of the binary upload and not part of the repository.  A test is included in the PR to ensure that the link works.  This applies to all links with the class "dl"---none of them are expected to exist in local builds, all of them are expected to exist on the live server, and all of them are checked by the test included in this PR.